docs/source/rest.html.md.erb

```diff
@@ -95,7 +95,7 @@ You can pass two letters country codes ([ISO 639-1](https://en.wikipedia.org/wik
 Type: string
 </td>
 <td markdown="1">
-If specified, restrict the search results to a specific list of comma-separated countries. You can pass two letters country codes ([ISO 639-1](https://en.wikipedia.org/wiki/ISO_3166-1#Officially_assigned_code_elements)).
+If specified, restrict the search results to a specific list of comma-separated countries. You can pass two letters country codes ([ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1#Officially_assigned_code_elements)).
```

Comment:
Not sure here, we are now linking two different pages. You can go for just https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes. Thanks